From linux-kernel@vger.kernel.org Tue Jan 13 00:59:09 2009
X-Envelope-From: <git-commits-head-owner@vger.kernel.org>
X-Envelope-To: <vger@sl-h.de>
X-Delivery-Time: 1231804755
X-UID: 403519
Return-Path: <git-commits-head-owner@vger.kernel.org>
X-RZG-CLASS-ID: mi
Received: from vger.kernel.org ([209.132.176.167])
	by mailin.webmailer.de (lemon mi35) (RZmta 18.6)
	with ESMTP id r0292dl0CNEDGl for <vger@sl-h.de>;
	Tue, 13 Jan 2009 00:59:14 +0100 (MET)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1753179AbZALX7N (ORCPT <rfc822;vger@sl-h.de>);
	Mon, 12 Jan 2009 18:59:13 -0500
Received: (majordomo@vger.kernel.org) by vger.kernel.org id S1754341AbZALX7N
	(ORCPT <rfc822;git-commits-head-outgoing>);
	Mon, 12 Jan 2009 18:59:13 -0500
Received: from hera.kernel.org ([140.211.167.34]:33094 "EHLO hera.kernel.org"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1753577AbZALX7M (ORCPT
	<rfc822;git-commits-head@vger.kernel.org>);
	Mon, 12 Jan 2009 18:59:12 -0500
Received: from hera.kernel.org (IDENT:U2FsdGVkX18hX4wxd9T/KRFI+O8DtieyPHv8umz2iH4@localhost [127.0.0.1])
	by hera.kernel.org (8.14.2/8.14.2) with ESMTP id n0CNxA9N015555
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NO)
	for <git-commits-head@vger.kernel.org>; Mon, 12 Jan 2009 23:59:10 GMT
Received: (from dwmw2@localhost)
	by hera.kernel.org (8.14.2/8.13.1/Submit) id n0CNx9rO015537
	for git-commits-head@vger.kernel.org; Mon, 12 Jan 2009 23:59:09 GMT
Date:	Mon, 12 Jan 2009 23:59:09 GMT
Message-Id: <200901122359.n0CNx9rO015537@hera.kernel.org>
From:	Linux Kernel Mailing List <linux-kernel@vger.kernel.org>
To: git-commits-head@vger.kernel.org
Subject: ALSA: hda - Fix typos for AD1882 codecs
MIME-Version: 1.0
Content-Type: text/plain;
  charset=UTF-8
Content-Transfer-Encoding: 8bit
X-Git-Commit: c247ed6f5205f9feebd276c4cbe45018b10f19fa
X-Git-Parent: d5337debacc00591b3f81fc3c982b40af7de1ab6
X-Virus-Scanned: ClamAV 0.93.3/8856/Mon Jan 12 16:36:19 2009 on hera.kernel.org
X-Virus-Status:	Clean
X-Spam-Status: No, score=-2.6 required=5.0 tests=AWL,BAYES_00,
	UNPARSEABLE_RELAY autolearn=ham version=3.2.5
X-Spam-Checker-Version:	SpamAssassin 3.2.5 (2008-06-10) on hera.kernel.org
X-Greylist: Sender IP whitelisted, not delayed by milter-greylist-4.0 (hera.kernel.org [127.0.0.1]); Mon, 12 Jan 2009 23:59:11 +0000 (UTC)
Sender:	git-commits-head-owner@vger.kernel.org
Precedence: bulk
List-ID: <git-commits-head.vger.kernel.org>
X-Mailing-List:	git-commits-head@vger.kernel.org
Status: R
X-Status: NC
X-KMail-EncryptionState:  
X-KMail-SignatureState:  
X-KMail-MDN-Sent:  

Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=c247ed6f5205f9feebd276c4cbe45018b10f19fa
Commit:     c247ed6f5205f9feebd276c4cbe45018b10f19fa
Parent:     d5337debacc00591b3f81fc3c982b40af7de1ab6
Author:     Clemens Fruhwirth <clemens@endorphin.org>
AuthorDate: Wed Jan 7 11:43:48 2009 +0100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Wed Jan 7 11:43:48 2009 +0100

    ALSA: hda - Fix typos for AD1882 codecs
    
    Fixed typos of codec-id checks for AD1882/AD1882A.
    
    Cc: stable@kernel.org
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 sound/pci/hda/patch_analog.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/pci/hda/patch_analog.c b/sound/pci/hda/patch_analog.c
index 13e4d91..2e7371e 100644
--- a/sound/pci/hda/patch_analog.c
+++ b/sound/pci/hda/patch_analog.c
@@ -4263,13 +4263,13 @@ static int patch_ad1882(struct hda_codec *codec)
 	spec->num_adc_nids = ARRAY_SIZE(ad1882_adc_nids);
 	spec->adc_nids = ad1882_adc_nids;
 	spec->capsrc_nids = ad1882_capsrc_nids;
-	if (codec->vendor_id == 0x11d1882)
+	if (codec->vendor_id == 0x11d41882)
 		spec->input_mux = &ad1882_capture_source;
 	else
 		spec->input_mux = &ad1882a_capture_source;
 	spec->num_mixers = 2;
 	spec->mixers[0] = ad1882_base_mixers;
-	if (codec->vendor_id == 0x11d1882)
+	if (codec->vendor_id == 0x11d41882)
 		spec->mixers[1] = ad1882_loopback_mixers;
 	else
 		spec->mixers[1] = ad1882a_loopback_mixers;
--
To unsubscribe from this list: send the line "unsubscribe git-commits-head" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

